iNaturalist/src/main/java/org/inaturalist/android/INaturalistApp.java

```diff
@@ -1143,9 +1146,9 @@ public String getCurrentTimeZoneName() {
     public String shortFormatDate(Timestamp date) {
         SimpleDateFormat f;
         if (Locale.getDefault().getCountry().equals("US")) {
-            f = new SimpleDateFormat("MMM d, yyyy");
+            f = new SimpleDateFormat("MMM d, yyyy", mLocale);
         } else {
-            f = SHORT_DATE_FORMAT;
+            f = new SimpleDateFormat("d MMM yyyy", mLocale);
```

Comment:
Let's globalize this as `date_long` or something, or just using the existing `date_short`. IMO, we should do that for the US too.